android/app/src/googleplay/java/org/fossasia/openevent/fragments/MapsFragment.java

```diff
@@ -180,42 +176,41 @@ private void showLocationsOnMap() {
         });
     }
 
-    private void focucOnMarker(Marker marker){
-        marker.showInfoWindow();
```

Comment:
@shailesh351 tried this before. The thing is in that case, the search only works if the marker is visible. If it is clustered, then the app crashes since the cache is cleared in that case.